pkg/controller/v1alpha2/trial/trial_controller.go

```diff
@@ -54,15 +65,28 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Create a new controller
 	c, err := controller.New("trial-controller", mgr, controller.Options{Reconciler: r})
 	if err != nil {
+		log.Error(err, "Create trial controller error")
 		return err
 	}
 
 	// Watch for changes to Trial
 	err = c.Watch(&source.Kind{Type: &trialsv1alpha2.Trial{}}, &handler.EnqueueRequestForObject{})
 	if err != nil {
+		log.Error(err, "Trial watch error")
+		return err
+	}
+
+	err = c.Watch(&source.Kind{Type: &batchv1.Job{}},
```

Comment:
miss other job type?

Reply:
Currently this PR handles only default Job. I will do other Job types in a separate PR